.dependabot/config.yml

```diff
@@ -13,12 +13,6 @@ update_configs:
           # These should be grouped. Maintainer advise: Watch https://github.com/babel/babel
           # for new releases and file a PR if you got time.
           dependency_name: '@babel/*'
-      - match:
-          # Can't upgrade while using `@material-ui/pickers@3.x`
-          dependency_name: '@date-io/core'
-      - match:
-          # Can't upgrade while using `@material-ui/pickers@3.x`
-          dependency_name: '@date-io/date-fns'
```

Comment:
We don't depend on them yet and don't expect this to happen until we reach v5 or which it's no longer an issue.